### DISCUSSION TITLE
Help Unstructure service

### DISCUSSION THREAD

**Discussion by youqugit** [langgenius/dify#8418] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Unstructured service is called through an API, so why do I need to pull the Unstructured image when deploying with Docker?

### 2. Additional context or comments

The Unstructured API is a paid service, but the locally deployed package is open-source. So, can I use the locally downloaded package with docker?

**Comment by mamema** [comment]
i´m also interested to know this.
Here: https://github.com/langgenius/dify/issues/9000#top
is mentioned that unstructured may not be used, but finaly unanswered.
I´ve configured unstructured docker container and changed the needed settings. I even exposed the port, which wasn´t/isn´t mentioned in the original compose file.
But according to the containber logs, i see no unstructered container usage.

Is this even used or not?
